app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    vaccine = True
+    masks_needed = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except (NotVaccinatedError, OutdatedVaccineError):
+            vaccine = False
+        except NotWearingMaskError:
+            masks_needed += 1
+
+    if vaccine is False:
+        return "All friends should be vaccinated"
```

Comment:
Could you return this in `except` clause? Then you won't need `vaccine` variable at all